.devops/ci.yml

```diff
@@ -0,0 +1,105 @@
+schedules:
+  - cron: "30 23 * * 6"
+    displayName: Build
+    always: true
+    branches:
+      include:
+        - dev
+
+pr: none
+trigger:
+  branches:
+    include:
+      - dev
+
+resources:
+  repositories:
+    - repository: 1ESPipelineTemplates
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+
+variables:
+  - group: 'appcat-variables'
+  - name: version
+    value: "3.8.8"
```

Comment:
same comment here about maven 3.9.9 and perhaps using mvnw

Reply:
@kthatipally to verify the Maven version.

@karianna Do you have an example using `mvnw`? 


Thanks.

Reply:
Yeah gctoolkit and Censum both use it, but you can probably backlog that for later.  Let's get this pipeline going

Reply:
@joe-braley , In the past we have not been able to build using >3.8.8, there were some artifacts that were failing to build with the new maven versions. We already have a mvnw in the repo which points to 3.8.8. See here for sample command: https://github.com/Azure/appcat-rulesets/blob/b79b7c72d84bf648a836596b709be30beb4262be/.github/workflows/pr_build.yml#L66.

Reply:
Might be worth a quick local attempt with 3.9.9

Reply:
@karianna I will create a backlog item and also one if we need to do this for the konveyor rulesets.